phplib/StringUtil.php

```diff
@@ -369,6 +369,15 @@ static function formatNumber($n, $decimals) {
     return number_format($n, $decimals, ',', '.');
   }
 
+    /**
+   * Returns the preposition needed for numerals ending with 20->99 
+   * @param string $amount
+   * @return string either ␣ or ␣de␣
+   */
+  static function getAmountPreposition($amount) {
+    $de = substr($amount, -2);
+    return ($de > 0 && $de < 20) ? "" : " de";
+  }
```

Comment:
:+1:  Ce atenție infinită la detalii! :-)

Reply:
Eh, ar mai trebui tratat și 0, respectiv singularul unei singure bucăți (1 definiție, nu 1 definiții). Prin extragerea paradigmei poate. :)

Reply:
tempnam(): file created in the system's temporary directory
Since this is just a notice and the function still works correctly, you can either place the "@" operator infront of the function call "@tempnam()" or set your personal temp path in your php.ini like this:
Asta îmi apărea mereu când modificam un fișier și Smarty recompila.